macosx.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# This script contains common code for building PyNE on various Debian-derived systems
+#
+
+function check_repo() {
+
+    repo_name=$1
+
+    if [ -d $repo_name ] ; then
+        read -p "Delete the existing $repo_name directory and all contents? (y/n) " -n 1 -r
+        if [[ $REPLY =~ ^[Yy]$ ]] ; then
+            rm -rf $repo_name
+        fi
+    fi
+
+}
+
+function build_moab {
+
+    # Install MOAB
+    cd ${install_dir}
+    mkdir -p moab
+    cd moab
+    check_repo moab-repo
+    git clone --branch Version5.1.0 --single-branch https://bitbucket.org/fathomteam/moab moab-repo
+    cd moab-repo
+    mkdir -p build
+    cd build
+    cmake ../ -DENABLE_HDF5=ON \
+              -DBUILD_SHARED_LIBS=ON \
+              -DENABLE_PYMOAB=ON \
+              -DENABLE_BLASLAPACK=OFF \
+              -DENABLE_FORTRAN=OFF \
+              -DCMAKE_INSTALL_PREFIX=${install_dir}/moab
+    make
+    make install
+    
```

Comment:
Difference from Ubuntu: why not update the ENV Variables directly here (instead of sourcing .bashrc below)?

Reply:
I think we should update the linux version accordingly:

I see 2 advantages:
- simplify the scripts
- allow to check if .bashrc if properly formed